SUSYBSMAnalysis/HSCP/test/AnalysisCode/Analysis_CommonFunction.h

```diff
@@ -495,12 +495,17 @@ void LoadDeDxCalibration(std::unordered_map<unsigned int,double>& TrackerGains,
 
 
 
-TH3F* loadDeDxTemplate(string path, bool splitByModuleType){
+TH3F* loadDeDxTemplate(string path, bool splitByModuleType, bool useStrip, bool usePixel){
```

Comment:
I don't want a "useStrip" or "usePixel" in dEdxTemplates.
Templates should contains everything always.  
Using pixel or strip probability should be handled when the templates are used